bin/asd-prereq-checker.sh

```diff
@@ -160,6 +161,12 @@ function check_project_id_is_valid {
 
 
 function check_quota_is_sufficient {
+  api=$(gcloud services list --format=json --filter=name:$COMPUTE_API)
+  if [[ "$api" != *"$COMPUTE_API"* ]]; then
+    echo "WARNING: Unable to verify compute quota because $COMPUTE_API in project $PROJECT_ID is not enabled. Enable this API in the current project at https://console.cloud.google.com/apis/api/compute.googleapis.com/overview?project=$PROJECT_ID, and run this script again."
```

Comment:
nit: the "," after the URL without any space is going to get copied as part of the URL. Can you remove the , after the URL even though it is technically gramatical?

Reply:
done